src/pages/AddProduct.js

```diff
@@ -2,6 +2,7 @@ import React, { useState } from 'react';
 import axios from 'axios';
 import './styles.css';
 import { useProductsContext } from "../context/products_context";
+import { ReactComponent as ArrowIcon } from '../assets/icons/arrow-with-circle.svg'
```

Comment:
We are able to import SVG as a React component because you're using the "Create React App" `react-scripts` package. If you weren't using that it'd require a bit of extra setup to do it this way.